newsletter/2020-05.md

```diff
@@ -63,8 +63,8 @@ similar to that of Google search.
 
 This AIP is a holdover from last month's newsletter: we intended to approve it
 in April, but several teams brought forth issues and clarifications that need
-to be made. We are working on getting these in, and hope to approve this AIP in
-May.
+to be made. These are now represented in the AIP, and hope to approve this AIP
```

Comment:
```suggestion
to be made. These are now represented in the AIP, and we hope to approve this AIP
```

Reply:
Done. I ended up changing the language slightly more.